app/Resources/views/team_interest/team_interest_receipt.html.twig

```diff
@@ -0,0 +1,40 @@
+<h1>Teaminteresse i Vektorprogrammet</h1>
+
+Denne mailen er et automatisk svar på din innmeldte teaminteresse.
+
+<p>
+    Hei {{ teamInterest.name }}!
+    Så hyggelig at du er interessert i teamvervene i Vektorprogrammet!
+    Som teammedlem får du bli med i en sammensveiset og sosial gjeng med studenter fra alle mulige linjer og trinn.
+    Vi arrangerer mange sosiale sammenkomster for teamene våre, som fester, hytteturer, tacokvelder, julebord osv.
+    I tillegg byr vervene på verdifull erfaring som du kan ta med deg videre i både studier og arbeidsliv.
+</p>
+<p>
+    Nedenfor er en liste med teamene du er interessert i.
+    Du kan trykke inn på lenkene for å lese mer om hvert av dem (og ikke minst søke!).
+    Lederne i de respektive teamene vil kontakte deg med videre informasjon.
+</p>
+
+<ul>
+    {% for team in teamInterest.potentialTeams %}
+        <li>
+            <h3>{{ team.name }}</h3>
+            {{ team.shortDescription }}<br>
+            Link til teamet:
+            <a href="{{ url('team_page_department_team', {departmentCity: team.department, teamName: team.name }) }}">
+                {{ url('team_page_department_team', {departmentCity: team.department, teamName: team.name }) }}
+            </a><br>
+            E-post: <a href="mailto:{{ team.email }}">{{ team.email }}</a>
+        </li>
+    {% endfor %}
+</ul>
+
+<p>
+    Hvis du bestemmer deg for å søke, kan du gjøre dette via lenkene over. Vi håper du vil være med!
+</p>
+
+<br>
+<p>Med vennlig hilsen Vektorprogrammet {{ teamInterest.department }}</p>
+<a href="{{ url('home') }}">
+    <img src={{ asset('images/vektor.png') }}" alt="{{ url('home') }}">
```

Comment:
1. Mangler en `"` etter `src=`
2. Dette vil linke til `images/vektor.png`, og vil derfor ikke være en gyldig bildepath i epost. Kan bruke `absolute_url` for å få med hele urlen: `absolute_url(asset('images/vektor.png'))`